File: Python/Size_Empty.py
def get_size(self):
    size = 0
    for i in range(self.TABLE_SIZE):
        current = self.table[i]
        while current:
            size += 1
            current = current.next
    return size

def is_empty(self):
    for i in range(self.TABLE_SIZE):
        if self.table[i] is not None:
            return False  # Not empty
    return True  # Empty

File: Size_Empty.c
int getSize(const HashMap* map) {
    int size = 0;
    for (int i = 0; i < TABLE_SIZE; i++) {
        KeyValue* current = map->table[i];
        while (current) {
            size++;
            current = current->next;
        }
    }
    return size;
}

int isEmpty(const HashMap* map) {
    for (int i = 0; i < TABLE_SIZE; i++) {
        if (map->table[i] != NULL) {
            return 0; // Not empty
        }
    }
    return 1; // Empty
}
